backstop/backstop.js

```diff
@@ -1,5 +1,9 @@
 const scenarios = require('./scenarios');
 
+// Using diferent Cypress image for M1 processors
+const isM1 = process.argv.includes('--m1');
+const dockerImage = isM1 ? `dockerman33/backstopjs:5.4.4` : `backstopjs/backstopjs:{version}`;
```

Comment:
Two things:
- this is using an older version of backstop (which includes a different version of puppeteer/chromium)
- shouldn't we build our own docker image instead of trusting some random person's?

Reply:
Link where I found Docker image: https://github.com/garris/BackstopJS/issues/1300#issuecomment-1069146987